Forms/index.html

```diff
@@ -9,82 +9,96 @@
 </head>
 
 <body>
-    <img src="res/left-img.png" class="img1">
-    <h1>
-        <font size="15" face="segoe print">Алко.net</font>
-    </h1>
+    <div class="head-container">
+        <h1>
+            <font size="15" face="segoe print">Алко.net</font>
+        </h1>
+        <img src="res/left-img.png" class="img1">
+    </div>
 
     <div class="container">
-        <form action="#" method="get" enctype="multipart/form-data" id="form">
-            <div class="first_collum">
+        <div class="first_column">
+            <div>Текст </div>
+        </div>
+        <form id="form">
+            <div class="second_column">
                 <p>
                 <h2>Мы проводим анкетирование, пожалуйста пройдите опрос</h2>
                 </p>
                 <div class="text_form">
                     <p>Введите имя</p>
-                    <input placeholder="Иванов Иван" type="text" name="username" value=""><!--тег disabled блокирует поле-->
+                    <input placeholder="Иванов Иван" type="text" name="username"><!--тег disabled блокирует поле-->
                 </div>
                 <div class="text_form">
                     <p>Введите email</p>
-                    <input tabindex="1" placeholder="simple_email@mail.com" type="text" name="userpass" value=""><!-- tabindex на скоьлко пунктов прескочит формы нажав tab-->
+                    <input tabindex="1" placeholder="simple_email@mail.com" type="text" name="userpass"><!-- tabindex на скоьлко пунктов прескочит формы нажав tab-->
                 </div>
             </div>
-        <br>
-        <div>
-            Выберите ваш тип пива:
-            <p>
-                <input checked type="radio" name="beer" value="lager">Свелое
-                <br>
-                <input type="radio" name="beer" value="dark">Тёмное
-            </p>
-        </div>
+            <br>
 
-        <div>
-            <p>Выберите любымый(ые) бренд(ы) пива</p>
-            <p>
-                <input type="checkbox" name="box" value="">Corona
-                <br>
-                <input type="checkbox" name="box1" value="">Alivaria
-                <br>
-                <input type="checkbox" name="box2" value="">Weihenstephaner
-                <br>
-                <input type="checkbox" name="box3" value="">Paulaner
-                <br>
-                <input type="checkbox" name="box4" value="">Жигулёвское
-            </p>
-        </div>
-        <div>
-            Выберите фото вашего любимого пипа:<br>
-            <p>
-                <input type="file" name="img" value=""><!--Выбор файла-->
-            </p>
-        </div>
+            <div class="form">
```

Comment:
The same as here https://github.com/iliasen/study_web/pull/1/files/ed2234ea9f81a52e84159bdb2a3b39e69a18730c..21513662b70c1f16945a3de24830944bf0eb674b#r1139756836